conanfile.py

```diff
@@ -102,17 +115,26 @@ def requirements(self):
             self.requires("poco/1.9.4")
 
         if self.options.with_qt:
-            self.options["sdl2"].fPIC = True
+            if self.settings.compiler != 'Visual Studio':
+                self.options["sdl2"].fPIC = True
             self.requires("qt/5.15.1@bincrafters/stable")
             self.requires("libjpeg/9d")
-
+        
+        if self.options.bites and self.settings.compiler != 'Visual Studio':
+            self.options["sdl2"].fPIC = True
 
         if self.options.with_cg:
             self.requires("nvidia-cg-toolkit-binaries/3.1.0013@utopia/testing")
 
         if self.settings.os == "Linux" and self.options.bites:
             self.requires("libxaw/1.0.13@bincrafters/stable")
 
+        if self.options.codec_freeimage:
+            self.requires("freeimage/3.18.0@utopia/testing")
+
+        #if self.options.opengles_renderer:
+            #self.requires("opengl/system") # It seems that the conan package is corrupted, nothing is generated...
```

Comment:
I believe the purpose of this package is to install system packages for opengl. It installs opengl headers GL/gl.h and also libraries libGL.so (on ubuntu at least). I don't think you should remove it. https://conan.io/center/opengl?tab=recipe